rust/lance-encoding/src/encodings/physical/bitmap.rs

```diff
@@ -189,8 +181,8 @@ mod tests {
                 },
             ],
         };
-        let mut dest = vec![BytesMut::with_capacity(1)];
-        let result = decoder.decode_into(5, 1, &mut dest);
+
+        let result = decoder.decode(5, 1, &mut false);
```

Comment:
I think we can now change this from a mutable input to a part of the output.  Let's fix in a future PR.